org.eclipse.m2e.core.ui/plugin.xml

```diff
@@ -674,4 +686,31 @@
    <extension point="org.eclipse.m2e.core.archetypeCatalogs">
       <remote  url="https://repo1.maven.org/maven2/archetype-catalog.xml" description="Maven Central"/>
    </extension>
+   <extension
+         point="org.eclipse.ui.menus">
+      <menuContribution
+            allPopups="false"
+            locationURI="menu:project">
+            <command
+                  commandId="org.eclipse.m2e.core.ui.command.updateProject"
+                  style="push">
```

Comment:
Another label should be used because the command label is "Update Project" (it's usually enough as we're in a Maven submenu). Here, without Maven context, it should be "Update Maven Project".
Or the other way round: the command should be relabeld to "Update Maven Project" (so the label also shows in Preference keys and Ctrl+3), and the label overriden only under "Maven" submenu.